content/about/feature-stages/index.md

```diff
@@ -70,13 +70,13 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Stable
 | [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/)        | Stable
-| [Kubernetes: Service Credential Distribution](/docs/concepts/security/mutual-tls/)   | Stable
+| [Kubernetes: Service Credential Distribution](/docs/concepts/security/#authentication)   | Stable
 | [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/plugin-ca-cert/)        | Stable
-| [Service-to-service mutual TLS](/docs/concepts/security/mutual-tls/)         | Stable
-| [Authentication policy](/docs/concepts/security/authn-policy/)  | Alpha
-| [VM: Service Credential Distribution](/docs/concepts/security/mutual-tls/)         | Beta
+| [Service-to-service mutual TLS](/docs/concepts/security/#authentication)         | Stable
+| [Authentication policy](/docs/concepts/security/#authorization)  | Alpha
```

Comment:
#authorization => authn-policy.

Let's leave "authn-policy" as is right now. We are going through reorganization of our security doc site. These names can be updated later. 

And we also plan to rename "Istio RBAC" to "Istio Authorization Policy".

Reply:
The name of the anchor (#authorization) is derive from the name of the header in the markdown. We can't have a section called "Authn Policy", which is why I changed it to authorization. Do you want me to call this Authorization Policy instead?

Reply:
It's actually authentication policy, not authorization policy. Istio RBAC is the authorization policy.